README.md

```diff
@@ -47,59 +43,55 @@ storage-component.googleapis.com
 ```bash
 python3 -m venv venv
 source ./venv/bin/activate
-pip install ml-pipeline-gen
+pip3 install ml-pipeline-gen
```

Comment:
We're using a virtualenv so `pip` should be set correctly already, no need to specify `pip3`.

```sh
❯ source venv/bin/activate
❯ pip -V
pip 19.2.3 from /usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv/lib/python3.7/site-packages/pip (python 3.7)
❯ which pip
/usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv/bin/pip
❯ deactivate
❯ source venv2/bin/activate
❯ pip -V
pip 20.1.1 from /usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv2/local/lib/python2.7/site-packages/pip (python 2.7)
❯ which pip
/usr/local/google/home/humichael/Documents/forks/ml-pipeline-generator-python/venv2/bin/pip
```